docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/tag/tag.stories.js

```diff
@@ -33,10 +33,9 @@ export const Tag = (knobTab) => {
   };
 
   const iconKnobTab = 'Icon';
-  const { icons } = ICONS;
   const withIcon = boolean('With icon', false, iconKnobTab);
   const iconKnobs = {
-    icon: withIcon ? select('Icon', icons, 'business_calendar', iconKnobTab) : null,
+    icon: withIcon ? select('Icon', Object.values(ICONS), 'calendar-1', iconKnobTab) : null,
```

Comment:
please do not use hardcoded value for `calendar-1` - instead, please use the very first array element of `ICONS`

Reply:
Updated all instances to use first element